README.md

```diff
@@ -1,9 +1,191 @@
 MICM Chemistry
 ==============
 
-Model Independent Chemical Mechanisms.
+Model Independent Chemical Module. MICM can be used to configure and solve atmospheric chemistry systems.
 
 [![License](https://img.shields.io/github/license/NCAR/micm.svg)](https://github.com/NCAR/micm/blob/master/LICENSE)
 [![CI Status](https://github.com/NCAR/micm/actions/workflows/test.yml/badge.svg)](https://github.com/NCAR/micm/actions/workflows/test.yml)
+[![codecov](https://codecov.io/gh/NCAR/micm/branch/main/graph/badge.svg?token=ATGO4DKTMY)](https://codecov.io/gh/NCAR/micm)
 
-Copyright (C) 2018-2020 National Center for Atmospheric Research
+Copyright (C) 2018-2023 National Center for Atmospheric Research
+
```

Comment:
maybe add a comment here to state that MICM versions 3.x.x are part of a refactor and that there may be non-backwards-compatible changes across minor versions?